narwhal/node/src/lib.rs

```diff
@@ -61,8 +61,8 @@ impl Node {
     where
         State: ExecutionState + Send + Sync + 'static,
     {
-        let initial_committee = ReconfigureNotification::NewEpoch((**committee.load()).clone());
-        let (tx_reconfigure, _rx_reconfigure) = watch::channel(initial_committee);
+        let initial_committee = ShutdownNotification::Run;
+        let (tx_shutdown, _rx_shutdown) = watch::channel(initial_committee);
```

Comment:
will there be a need to read the committee from the channel when it is now only being used as shutdown?